src/main/controllers/CaseDocumentController.ts

```diff
@@ -41,14 +40,14 @@ export default class CaseDocumentController {
 
       if (!details.mimeType) {
         res.setHeader('Content-Type', 'application/pdf');
-        logger.log('Failed document name: ' + details.originalDocumentName);
+        logger.warn('Failed document name: ' + details.id);
```

Comment:
Should this be a warn or an error? also we're losing the name, is this not important?

Reply:
The name can be PII, I'm not bothered whether it's a warn or error, if you feel strongly I can change.

Reply:
ah that makes sense, yeah I'm honestly not sure on warn or error, I'll leave to you to decide, my only thought was that the operation has failed if it failed to create a document

Reply:
On another thought, I will change it to error.
But on a side note, this _should_ never happen.